CMakeLists.txt

```diff
@@ -608,7 +608,7 @@ else()
   add_subdirectory(Externals/liblzma)
 endif()
 
-find_package(zstd)
+pkg_search_module(ZSTD QUIET libzstd)
```

Comment:
I'd prefer if we provide a Findzstd.cmake helper instead.

Reply:
Why?